### PR TITLE
Add License information

### PR DESCRIPTION
**Pull request by sudoshweta**
License information is added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,7 @@
 This repository will eventually house the source material for the documentation for the [OpenTracing](http://opentracing.io/documentation/) project.
 
 For the time being, that source material can be found in [this repository](https://github.com/opentracing/opentracing.io/tree/master/_docs).
+
+## License
+
+By contributing to OpenTracing documentation-1 repository, you agree that your contributions will be licensed under its [Apache 2.0 License]
```
Comment (jpkrohling) [inline README.md]: > documentation-1

Shouldn't it be just `documentation`?

File: README.md [hunk 2]
```diff
@@ -3,3 +3,7 @@
 This repository will eventually house the source material for the documentation for the [OpenTracing](http://opentracing.io/documentation/) project.
 
 For the time being, that source material can be found in [this repository](https://github.com/opentracing/opentracing.io/tree/master/_docs).
+
+## License
+
+By contributing to OpenTracing documentation repository, you agree that your contributions will be licensed under its [Apache 2.0 License]
```
Comment (jpkrohling) [inline README.md]: Looks like the reference to the license isn't being rendered as a link:

image

Comment (sudoshweta) [reply README.md]: Okay, I will change it.

File: README.md [hunk 3]
```diff
@@ -3,3 +3,7 @@
 This repository will eventually house the source material for the documentation for the [OpenTracing](http://opentracing.io/documentation/) project.
 
 For the time being, that source material can be found in [this repository](https://github.com/opentracing/opentracing.io/tree/master/_docs).
+
+## License
+
+By contributing to OpenTracing documentation repository, you agree that your contributions will be licensed under its Apache 2.0 License.
```
Comment (jpkrohling) [inline README.md]: You can add a link by specifying the link text within `[]` and the URL within `()`. As the `LICENSE` file is at the same directory as the readme file, the URL can be relative. Like this:

```
... will be licensed under its [Apache 2.0 License](./LICENSE).
```

**Review by jpkrohling — APPROVED** [review]

**Comment by yurishkuro** [comment]
thx

**Pull request merged**